code/modules/mob/living/simple_animal/hostile/thing.dm

```diff
@@ -0,0 +1,1381 @@
+		animation.icon = 'icons/mob/mob.dmi'
+		animation.icon_state = "serf_appear"
+		animation.layer = 4
+		animation.desc = "What..."
+		animation.master = holder
+		animation.dir = dir
+		flick("serf_appear",animation)
+		visible_message("<span class='warning'>\the [src] rises from the creep!</span>")
+		anchored = 1
+		sleep(14)
+		if(animation)
+			forceMove(animation.loc)
+			qdel(animation)
+			QDEL_NULL(holder)
+			alpha = 255
+			burrowing = 0
+			anchored = 0
+			canmove = 1
+		return
+
+/obj/effect/dummy/serf
+	name = "???"
+	desc = "Something is moving underground..."
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "nothing"
+	density = 0
+	anchored = 1
+	invisibility = 60
+	burn_state = LAVA_PROOF
+
+/obj/effect/dummy/serf/relaymove(mob/user, direction)
+	forceMove(get_step(src,direction))
+	user.dir = direction
+
+/obj/effect/dummy/serf/ex_act()
+	return
+
+/obj/effect/dummy/serf/bullet_act()
+	return
+
+/obj/effect/dummy/serf/singularity_act()
+	return
+
+/mob/living/simple_animal/hostile/serf/death()
+	. = ..()
+	if(!.)
+		return FALSE
+
+	var/turf/E = get_turf(src)
+	var/atom/movable/overlay/animation = new /atom/movable/overlay(E)
+	qdel(src)
+	animation.name = "???"
+	animation.density = 0
+	animation.anchored = 1
+	animation.icon = 'icons/mob/mob.dmi'
+	animation.icon_state = "serf_die"
+	animation.layer = 4
+	animation.desc = "What..."
+	animation.master = holder
+	animation.dir = dir
+	flick("serf_die",animation)
+	sleep(14)
+	if(animation)
+		qdel(animation)
+
+/mob/living/simple_animal/hostile/serf/Stat()
+	..()
+	if(statpanel("Status"))
+		stat(null, "Time survived: [life].")
+
+/mob/living/simple_animal/hostile/serf/Life()
+	. = ..()
+	if(stat != DEAD)
+		life++
+		dissolve_cooldown--
+		appear_cooldown--
+		if(life < 600) // Newly spawned
+			maxHealth = 150
+			desc = "An impossible being that breaks all concepts of reality just by existing."
+			melee_damage_lower = 20
+			melee_damage_upper = 25
+			color = ""
+			environment_smash = 1
+		if(life >= 600) // Been alive for some time, let's give you more health.
+			maxHealth = 200
+			desc = "An impossible being that breaks all concepts of reality just by existing. This one has existed for some time."
+			melee_damage_lower = 20
+			melee_damage_upper = 25
+			color = ""
+			environment_smash = 1
+		if(life >= 1200) // How the heck are you still alive. Let's make it easier for you to survive, then.
+			maxHealth = 250
+			desc = "An impossible being that breaks all concepts of reality just by existing. This one has existed for a long time."
+			melee_damage_lower = 25
+			melee_damage_upper = 35
+			color = ""
+			environment_smash = 1
+		if(life >= 2000) // Last upgrade.
+			maxHealth = 300
+			desc = "An impossible being that breaks all concepts of reality just by existing. This one is ancient."
+			melee_damage_lower = 30
+			melee_damage_upper = 35
+			color = "#e5e5e5" // Becomes ever so slightly different.
+			environment_smash = 2
+		if(health != maxHealth)
```

Comment:
This should be a < rather than !=, otherwise you risk things getting out of hand